pkg/server/rest/v2/server.go

```diff
@@ -0,0 +1,49 @@
+package rest
```

Comment:
Can you add an attribution comment to Envoyproxy Authors for consistency and a one line package comment for godoc?